scripts/build_rust_package.sh

```diff
@@ -50,10 +50,10 @@ build_rust_package() {
                CARGO_TERM_COLOR='never'
         cd '/build/$package'
         cargo fetch
-        cargo build --release --frozen ${extra_features:+--features $extra_features}
+        cargo build -p $package --release --frozen ${extra_features:+--features $extra_features}
```

Comment:
I have more than one crate in the workspace, so i need to specify which one i want to build.  Im not sure whether this change would effect other users of this script - need to check their crate names (lighthouse, reth and rbuilder).